transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystemRootResourceDefinition.java

```diff
@@ -281,6 +281,13 @@
             .setAllowExpression(true)
             .setRequires(CommonAttributes.USE_JDBC_STORE).build();
 
+    public static final SimpleAttributeDefinition STALE_TRANSACTION_TIME = new SimpleAttributeDefinitionBuilder(CommonAttributes.STALE_TRANSACTION_TIME, ModelType.INT, true)
+            .setValidator(new IntRangeValidator(0))
+            .setMeasurementUnit(MeasurementUnit.SECONDS)
+            .setDefaultValue(new ModelNode().set(600))
+            .setFlags(AttributeAccess.Flag.RESTART_NONE)
```

Comment:
This should be RESTART_ALL_SERVICES -- ReloadRequiredWriteAttributeHandler handles the writes.

Reply:
fixed